Source/Core/DolphinWX/Config/WiiConfigPane.cpp

```diff
@@ -83,8 +88,15 @@ void WiiConfigPane::InitializeGUI()
 
 void WiiConfigPane::LoadGUIValues()
 {
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bProgressive) {
```

Comment:
Either put the braces on a new line, or omit them since its just a single-line statement